File: INTRODUCCION A POO/Ejercicio1/codigo/e1.java
public class Persona {
    private String nombre;
    private int edad;
    private String ciudad;

    public Persona(String nombre, int edad, String ciudad) {
        this.nombre = nombre;
        this.edad = edad;
        this.ciudad = ciudad;
    }

    public void saludo() {
        System.out.println("Hola soy " + nombre + " de " + ciudad);
    }

    public void medad() {
        if (edad >= 18) {
            System.out.println(nombre + " es mayor de edad");
        } else {
            System.out.println(nombre + " es menor de edad");
        }
    }

    public static void main(String[] args) {
        Persona persona1 = new Persona("Ronald", 17, "La Paz");
        Persona persona2 = new Persona("Juan", 15, "La Paz");
        Persona persona3 = new Persona("Jose", 19, "La Paz");

        persona1.saludo();
        persona1.medad();
        persona2.saludo();
        persona3.saludo();
    }
}

File: INTRODUCCION A POO/Ejercicio1/codigo/e1.py
#1. Crea una clase Persona con nombre, edad y ciudad
#a) Agrega un método para mostrar el saludo: “Hola, soy {nombre} de {ciudad}”
#b) Crea tres personas y muestra su saludo
#c) Agrega un método para verificar si es mayor de edad

class Persona:
    def __init__(self,nombre,edad,ciudad):
        self.nombre = nombre
        self.edad = edad
        self.ciudad = ciudad

    def saludo(self):
        print("Hola soy ",self.nombre, " de ", self.ciudad)
    
    def medad(self):
        if self.edad >=18:
            print (self.nombre,"Es mayor de edad")
        else:
            print (self.nombre, "Es menor de edad")

Persona1 = Persona("Ronald", 17 , "La Paz")
Persona2 = Persona("Juan", 15 , "La Paz")
Persona3 = Persona("Jose", 19 , "La Paz")
Persona1.saludo()
Persona1.medad()
Persona2.saludo()
Persona3.saludo()
